apps/src/templates/sectionAssessments/StudentsMCSummaryContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React, {Component, PropTypes} from 'react';
+import StudentsAssessmentsMCTable, {studentOverviewDataPropType} from './StudentsAssessmentsMCTable';
+import {
+  getStudentsMCSummaryForCurrentAssessment,
+} from './sectionAssessmentsRedux';
+import {connect} from 'react-redux';
+
+class StudentsMCSummaryContainer extends Component {
+  static propTypes = {
+    studentOverviewData: PropTypes.arrayOf(studentOverviewDataPropType),
+  };
+
+  render() {
+    return (
+      <div>
+        {/* Temporary placeholder for the header of the table */}
```

Comment:
Normally if I have a comment about something that will change in the future, I format like a TODO so that I'm definitely accountable to come back and fix it. In this case it's pretty obvious so no need to update, but just an FYI.